### ISSUE TITLE
Implement seeding scripts to create data for testing/development

### ISSUE THREAD

**Issue by mushketyk** [FightPandemics/FightPandemics#194]
Implement a seeding script that should create projects and organizations.

We can use Mongo Seeding project for this: https://github.com/pkosiec/mongo-seeding

**Comment by mushketyk** [comment]
Duplicate of #163

**Comment by joshmorel** [comment]
@mushketyk - on further consideration I don't think this is a duplicate of #163. #163 is about importing data sourced by the data collection team into the production database mongo both one-time and as part of incremental load. It could be used for development as well but separating this makes sense I think due to the different purpose.

This would be for seeding databases for all collections (not just userd/posts) with dummy data for development & testing.

I did start my work on the other ticket with mongo-seeding and was able to get it to work fine - until I needed to asynchronously pull data from airtable. I've decided to switch to mongodb native driver (eventually mongoose) for that but for this mongo-seeding should work.

**Comment by indirabhowmick** [comment]
@joshmorel - Should I assign the ticket to you?

**Comment by joshmorel** [comment]
@indirabhowmick I assigned myself. This isn't required for MVP but developer productivity. it is different from #408 as it would be "dummy" data for dev/test of more than just posts - users, comments etc. I will work on it after #408 is done because post imports from Airtable might be enough for now for devs working on MVP.

**Comment by indirabhowmick** [comment]
@joshmorel - Thanks so much for the detailed explanation
